rust/src/fvm/machine.rs

```diff
@@ -207,6 +218,22 @@ pub unsafe extern "C" fn fil_fvm_machine_execute_message(
             }
         };
 
+        if !apply_ret.exec_trace.is_empty() {
+            let mut trace_iter = apply_ret.exec_trace.iter();
```

Comment:
```suggestion
            let mut trace_iter = apply_ret.exec_trace.into_iter();
```

will save you some clones later